### PR TITLE
[CLOUDTRUST-2298] Add fields to realm configuration

### PR DESCRIPTION
**Pull request by fperot74**
Requires:
* cloudtrust/common-service#38

### CODE REVIEW

File: pkg/account/authorization.go [hunk 1]
```diff
@@ -141,18 +141,26 @@ func (c *authorizationComponentMW) UpdateAccount(ctx context.Context, account ap
 		return err
 	}
 
+	if !isEnabled(config.APISelfAccountEditingEnabled) {
```
Comment (harture) [inline pkg/account/authorization.go]: I don't see the difference of behavior between APISelfAccountEditingEnabled and APISelfMailEditingEnabled. Both needs to be enabled to be able to perform an edition. But it is not possible to allow to only edit the email. Should we enhance our mechanism to be able to provide finer grain of authorization for edition or remove the falg related to mail edition ?

Comment (fperot74) [reply pkg/account/authorization.go]: Removing APISelfMailEditingEnabled

**Comment by fperot74** [comment]
> problem though : The email sent when changing the phone number doesn't let me va

I just retested this. I clicked on the link I received by mail. Received an SMS code and enter it to succesfully verify my phone number

File: pkg/management/component.go [hunk 2]
```diff
@@ -1178,6 +1178,7 @@ func (c *component) GetRealmCustomConfiguration(ctx context.Context, realmName s
 				APISelfAccountDeletionEnabled:       &falseBool,
 				ShowAuthenticatorsTab:               &falseBool,
 				ShowPasswordTab:                     &falseBool,
+				ShowProfileTab:                      &falseBool,
 				ShowMailEditing:                     &falseBool,
```
Comment (harture) [inline pkg/management/component.go]: ShowMailEditing should not exist anymore

File: pkg/management/component.go [hunk 3]
```diff
@@ -1199,6 +1200,7 @@ func (c *component) GetRealmCustomConfiguration(ctx context.Context, realmName s
 		APISelfAccountDeletionEnabled:       config.APISelfAccountDeletionEnabled,
 		ShowAuthenticatorsTab:               config.ShowAuthenticatorsTab,
 		ShowPasswordTab:                     config.ShowPasswordTab,
+		ShowProfileTab:                      config.ShowProfileTab,
 		ShowMailEditing:                     config.ShowMailEditing,
```
Comment (harture) [inline pkg/management/component.go]: same here

File: pkg/management/component.go [hunk 4]
```diff
@@ -1270,6 +1272,7 @@ func (c *component) UpdateRealmCustomConfiguration(ctx context.Context, realmNam
 		APISelfAccountDeletionEnabled:       customConfig.APISelfAccountDeletionEnabled,
 		ShowAuthenticatorsTab:               customConfig.ShowAuthenticatorsTab,
 		ShowPasswordTab:                     customConfig.ShowPasswordTab,
+		ShowProfileTab:                      customConfig.ShowProfileTab,
 		ShowMailEditing:                     customConfig.ShowMailEditing,
```
Comment (harture) [inline pkg/management/component.go]: same here

File: pkg/management/component.go (hunk 2 above)
Comment (fperot74) [reply pkg/management/component.go]: Removed

**Pull request merged**